pom.xml

```diff
@@ -69,8 +69,6 @@
 
         <module>qa</module>
 
-        <module>distro/h2-webapp</module>
```

Comment:
We should also remove this module from the `distro-serverless` profile.

Reply:
Good catch, I removed it from there as well.